deerlab/dipolarbackground.py

```diff
@@ -48,73 +50,58 @@ def dipolarbackground(t,pathinfo,Bmodel,renormalize = True,overtonecoeff = 1):
         lam = 0.4 # modulation depth main signal
         conc = 200   # spin concentration (uM)
 
-        pathinfo = [[],[]]
-        pathinfo[0] = [1-lam, NaN] # unmodulated part, gives offset
-        pathinfo[1] = [lam, 0] # main modulation, refocusing at time zero
+        pathinfo = []

```

Comment:
Can you simplify this to `pathinfo = [[1-lam],[lam,0]]`? I don't think using `append` is the dominant use case.

Reply:
The main idea was to define each pathway individually more clearly. But I agree that `append` might not be the most common use.

Reply:
See 0f8017d